pages/common/msmtp.md

```diff
@@ -0,0 +1,16 @@
+# msmtp
+
+> An SMTP client.
+> In the default sendmail mode, msmtp reads a mail from standard input and sendscit to an SMTP server for delivery.
+
+- Send a mail using the default account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp {{to@example.org}}`
+
+- Send a mail using a specific account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp --account={{example}} {{to@example.org}}`
+
+- Send a mail without a configured account:
+
+`echo {{"Hello world"}} | msmtp --host={{localhost}} --port={{999}} --from={{from@example.org}} {{to@example.org}}`
```

Comment:
You are partially right. One can specify the authentication mode in the config file - http://msmtp.sourceforge.net/doc/msmtp.html#A-user-configuration-file. I think we should add some authentication options in the command line too, just to make it a complete example. What do you think ?